DabraHalleloEndaAbunaSamuel/DHAS016.xml

```diff
@@ -0,0 +1,124 @@
+            <publicationStmt>
+               <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+               <pubPlace>Hamburg</pubPlace>
+               <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                  Forschungsumgebung / Beta maṣāḥǝft</publisher>
+               <availability>
+                  <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
+                     This file is licensed under the Creative Commons Attribution-ShareAlike 4.0.
+                  </licence>
+               </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS1019DabraHalleloEndaSamuel"/>
+                        <idno>DHAS-016</idno>                    
+                        </msIdentifier>
+                    <msContents>
+                       <summary/>
+                        <msItem xml:id="ms_i1">
+                            
+                            <title type="complete" ref="LIT2358Sirach"> Book of Sirach</title>
+                            <textLang mainLang="gez"/>
+                        </msItem>     
+                        </msContents>
+                   <physDesc>
+                     <objectDesc form="Codex">
+                        <supportDesc>
+                           <support>                          
+                                           <material key="parchment"/>
+                                       </support>
+                                       <extent>                                          
+                                                       <measure unit="leaf"/>                                  
+                                                       <dimensions type="outer" unit="mm">
+                                                           <height cert="low">280</height>
+                                                           <width cert="low">230</width>
+                                                           <depth cert="low">50</depth>
+                                                           
+                                           </dimensions>
+                                       </extent>   
+                            
+                            <condition key="good"/>
+                            
+                         </supportDesc>
+            
+                         <layoutDesc>
+                             <layout columns="2" cert="high">                                 
+                             </layout>
+                         </layoutDesc>
+                         
+                     </objectDesc>
+                       <handDesc>
+                           <handNote xml:id="h1" script="Ethiopic">
+                               <seg type="script">19th-/early 20th-century script.</seg>
```

Comment:
19th/early 20th century script.

Reply:
Sorry, actually I used to formulate like this, always, and in all my descriptions.
The same in Dair as-Suryan catalogue which was proof read, quite carefully (only the numbers were written out). If it is not correct, perhaps a sample should be elaborated and introduced into the guidelines.

Reply:
OK. If it is common practice, I will not remark it any more in future.

Reply:
(As far as I know, the form with hyphens is correct English)